app/store/middleware/persistState.js

```diff
@@ -1,10 +1,20 @@
-import { compose } from 'redux';
-import persistState from 'redux-localstorage';
-import adapter from 'redux-localstorage/lib/adapters/localStorage';
-import filter from 'redux-localstorage-filter';
+export const loadPersistedLocale = () => {
+  try {
+    const locale = localStorage.getItem('userLocale');
```

Comment:
Do this file in general need to write unit tests.. ?

Reply:
That's a good question. IMO no.

Reply:
Im writing unit tests for it, better to have some even though i trust localStorage usage